source/guide_nextcloud.rst

```diff
@@ -132,6 +132,15 @@ Updates
 
 The easiest way to update Nextcloud is to use the web updater provided in the admin section of the Web Interface.
 
+If you have installed Nextcloud on a subdomain it can happen that the update fails: Access to the UI is not possible and HTTP 403 errors are thrown.
+In most cases this happens due to wrong owner rights which can be fixed with finishing the update via console and fixing the rights.
```

Comment:
The wording is a little off here. The ownership rights are perfectly fine, bug the [SELinux security labels](https://wiki.gentoo.org/wiki/SELinux/Labels#Introduction) are wrong. The `restorecon` makes sure the labels are set as they are defined in the loaded selinux policy.